scripts/uberenv/spack_configs/blueos_3_ppc64le_ib_p9/packages.yaml

```diff
@@ -0,0 +1,78 @@
+
+# -------------------------------------------------------------------------
+# This file controls default concretization preferences for Spack.
+#
+# Settings here are versioned with Spack and are intended to provide
+# sensible defaults out of the box. Spack maintainers should edit this
+# file to keep it current.
+#
+# Users can override these settings by editing the following files.
+#
+# Per-spack-instance settings (overrides defaults):
+#   $SPACK_ROOT/etc/spack/packages.yaml
+#
+# Per-user settings (overrides default and site settings):
+#   ~/.spack/packages.yaml
+# -------------------------------------------------------------------------
+packages:
+  all:
+    compiler: [gcc, intel, pgi, clang, xl, nag]
+    providers:
+      awk: [gawk]
+      blas: [openblas]
+      daal: [intel-daal]
+      elf: [elfutils]
+      golang: [gcc]
+      ipp: [intel-ipp]
+      java: [jdk]
+      lapack: [openblas]
+      mkl: [intel-mkl]
+      mpe: [mpe2]
+      mpi: [mvapich2]
+      opencl: [pocl]
+      openfoam: [openfoam-com, openfoam-org, foam-extend]
+      pil: [py-pillow]
+      scalapack: [netlib-scalapack]
+      szip: [libszip, libaec]
+      tbb: [intel-tbb]
+      jpeg: [libjpeg-turbo, libjpeg]
+# LLNL blueos CUDA
+  cuda:
+    paths:
+      cuda@10.1.168: /usr/tce/packages/cuda/cuda-10.1.168
+      cuda@9.2.148: /usr/tce/packages/cuda/cuda-9.2.148
+    buildable: False
+# LLNL blueos mpi
+# note, we need to fake this as an existing mpi package
+# so even though this is spectrum mpi, we call it mvapich2
+  mvapich2:
+    paths:
+      mvapich2@2.2%gcc@7.3.1: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-gcc-7.3.1/
+      mvapich2@2.2%clang@upstream_xlf: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-upstream-2019.03.26/
+      mvapich2@2.2%clang@upstream_nvcc_xlf: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-upstream-2019.03.26/
+      mvapich2@2.2%xl@coral : /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-upstream-2019.03.26/
+    buildable: False
+# System level packages to not build
+  bzip2:
+   paths:
+      bzip2: /usr/bin/
+   buildable: False
+  gettext:
+   paths:
+      gettext: /usr/bin/
+   buildable: False
+  m4:
+   paths:
+      m4: /usr/bin/
+   buildable: False
+  tar:
+   paths:
+      tar: /usr/bin/
+   buildable: False
+  graphviz:
+    paths:
+      graphviz: /usr/bin/
+    buildable: False
+  cmake:
+    version: [3.9.6]
```

Comment:
pkg-config has bled into the TPL build somehow. Don't bother rebuilding but add it to the buildable: False entries.

Reply:
Thanks @white238 -- done.